hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryConfig.java

```diff
@@ -75,6 +76,9 @@
     public InventoryConfig() {
         configPath = Paths.get(System.getProperty("jboss.server.config.dir"), "hawkular");
```

Comment:
while we are doing mkdirs in this method, perhaps we should also mkdirs() this hawkular directory just to be safe?

Reply:
Yes, we can protect here and perform a mkdirs() too.